app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -22,6 +22,7 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
             $scope.items = data.data.items;
             $scope.relations = data.data.relations;
             $scope.kinds = data.data.kinds;
+            $scope.icons = data.data.icons;
```

Comment:
why are the icons places on scope? does Angular need them? aren't they used internally here in the controller?